### ISSUE TITLE
Iframe with bad src causing timeouts since version 1.4.0

### ISSUE THREAD

**Issue by ChristianLJ** [pocketjoso/penthouse#257]
Hi!

I am getting the following error since version 1.4. It isn't a problem in version 1.3.0.

```javascript
CRITICAL CSS ERROR Error: Penthouse timed out after 300s. 
    at Timeout.<anonymous> (...\node_modules\penthouse\lib\core.js:250:20)
    at ontimeout (timers.js:466:11)
    at tryOnTimeout (timers.js:304:5)
    at Timer.listOnTimeout (timers.js:267:5)
```

My config looks like this:
```javascript
penthouse({
        css: './domain.dk/' + hash + '.min.css',
        url: page,
        width: 1920,
        height: 1200,
        renderWaitTime: 3500,
        timeout: 300000,
        strict: false, 
        blockJSRequests: false,
        skipErrors: false,
        forceInclude: genericForceInclude.concat(forceInclude),
    }).then(criticalCss => {
        callback(criticalCss, page);
    }).catch(err => {
        console.log("CRITICAL CSS ERROR", err);
    });
```

**Comment by ChristianLJ** [comment]
So i was too quick but i managed to fix the error.

The error appears when i have an iframe in the following format:

```html
  <iframe style="width:100%" width="560" height="400" id="youtube"
                            src="?rel=0&amp;controls=1&amp;showinfo=0"
                            frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

But changning the src to a valid URL everything works as expected.

```html
  <iframe style="width:100%" width="560" height="400" id="youtube"
                            src="https://google.com?rel=0&amp;controls=1&amp;showinfo=0"
                            frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Btw. i am using Node v9.7.0.

**Comment by pocketjoso** [comment]
Interesting, I will try to reproduce.

**Comment by pocketjoso** [comment]
I can reproduce, your page never loads. Indeed this worked before, it broke when Penthouse upgraded to [Puppeteer@1.0.0](https://github.com/GoogleChrome/puppeteer/releases/tag/v1.0.0). I'm guessing your iframe is not found and how Puppeteer now works it never resolves the Page.load promise.

Puppeteer has changed a bit since the 1.0.0 version, but Penthouse has been blocked from upgrading. In the next release however an upgrade should be possible, so I'll come back to revisit this then to see if this issue can be resolved.

**Comment by pocketjoso** [comment]
Looks like this was indeed resolved with the latest Puppeteer version, now available in latest penthouse (1.4.5), via #261

**Comment by pocketjoso** [comment]
If you test and somehow this fix does not work for you, please re-open the issue.
